views/add-extension-reason.html

```diff
@@ -15,6 +15,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
      <form method="post">
+      {% include "includes/csrf_token.html" %}
```

Comment:
Indentation slightly out

Reply:
my configuration only indents the lines I have touched

Reply:
ok, but the new line is one character to the left of the rest of the block

Reply:
done